### PR TITLE
Integration test: Checkout and build Faucet

### PR DESCRIPTION
**Pull request by Geod24**

### CODE REVIEW

**Review by kchulj — APPROVED** [review]
LGTM.

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -26,14 +27,39 @@ jobs:
         sudo apt-get update
         sudo apt-get install g++-9 libsodium-dev libsqlite3-dev
 
+    - name: Checkout faucet
+      uses: actions/checkout@v2
+      with:
+        repository: bosagora/faucet
```
Comment (Geod24) [inline .github/workflows/integration.yml]: So it turns out that actually checking out faucet instead of agora works better, who would've guessed.

Comment (kchulj) [reply .github/workflows/integration.yml]: Hmm, so it was not checking out without the `repository`?

Comment (Geod24) [reply .github/workflows/integration.yml]: It is checking out Agora (again)

**Comment by Geod24** [comment]
Okay, finally it works.

**Pull request merged**